### PR TITLE
Added etherscan transactions

### PR DESCRIPTION
**Pull request by lionellbriones**

### CODE REVIEW

File: app/src/components/WalletHistory/TxHistoryTable/TxHistoryTable.vue [hunk 1]
```diff
@@ -9,7 +9,7 @@
       hide-default-footer
     >
       <template v-slot:default="props">
-        <TransactionDetails v-for="transaction in props.items" :key="transaction.id" :transaction="transaction" />
+        <TransactionDetails v-for="(transaction, index) in props.items" :key="index" :transaction="transaction" />
```
Comment (chaitanyapotti) [inline app/src/components/WalletHistory/TxHistoryTable/TxHistoryTable.vue:12]: change the key to be something other than the index

File: app/src/utils/utils.js [hunk 2]
```diff
@@ -547,7 +547,7 @@ export const formatPastTx = (x, lowerCaseSelectedAddress) => {
     type: x.type,
     type_name: x.type_name,
     type_image_link: x.type_image_link,
-    transaction_hash: x.transaction_hash,
+    isEtherscan: x.isEtherscan,
```
Comment (chaitanyapotti) [inline app/src/utils/utils.js:550]: keep the transaction_hash too.

File: app/src/components/WalletHistory/TxHistoryTable/TxHistoryTable.vue (hunk 1 above)
Comment (lionellbriones) [reply app/src/components/WalletHistory/TxHistoryTable/TxHistoryTable.vue]: Done

File: app/src/utils/utils.js (hunk 2 above)
Comment (lionellbriones) [reply app/src/utils/utils.js:551]: Done

**Pull request merged**